geyser-plugin-manager/src/geyser_plugin_service.rs

```diff
@@ -143,6 +146,7 @@ impl GeyserPluginService {
             transaction_notifier,
             entry_notifier,
             block_metadata_notifier,
+            slot_status_notifier,
```

Comment:
Do we need to add a new field to hold `slot_status_notifier`? Could it be pulled out of `slot_status_observer` instead?

Reply:
The slot status observer is notified via The following, which does not have these new status.
I chose not change this struct in order to minimize side effects in other workflows.
GeyserPluginService is an internal service in the validator so we are free to change the implementations.

pub enum SlotNotification {
    OptimisticallyConfirmed(Slot),
    /// The (Slot, Parent Slot) pair for the slot frozen
    Frozen((Slot, Slot)),
    /// The (Slot, Parent Slot) pair for the root slot
    Root((Slot, Slot)),
}